Hide! Series

```diff
@@ -0,0 +1,9 @@
+{
+    "project": "Hide! Series",
+    "tags": [
+        "Hide"
```

Comment:
tag is too vague, please read the readme

Reply:
I have updated the tags.  Let me know if that works.

Reply:
All the NFTs name begin with "Hide" followed by the species "Squid", for example, then followed by the number "003", for example. For example, HideSquid003 or HideUnknown001.  There are no spaces in between.  Would the updated tags work or would I have to list all of them due to there being no space in between?